config/v1/types_infrastructure.go

```diff
@@ -1002,6 +1002,9 @@ type PowerVSPlatformStatus struct {
 	// Note: Currently only single-zone OCP clusters are supported
 	Zone string `json:"zone"`
 
+	// ResourceGroup is the resource group for new IBMCloud resources created for the cluster.
```

Comment:
Which components are going to consume this? We should make sure to document which are and aren't going to consume this.

Can we provide any regex validation or the resource group? How would a user know what their resource group is? We need to document the expected value here a bit

Is this field optional? It will need a tag to mark it optional or required

Reply:
Tried to address the comments PTAL!